src/pages/Appointments/AppointmentsPage.tsx

```diff
@@ -556,7 +558,12 @@ export default function AppointmentsPage({
       </div>
 
       {activeTab === "board" ? (
-        <ScrollArea>
+        <ScrollArea
+          className={cn(
+            `transition-all duration-200 ease-${isSidebarOpen ? "out" : "in"}`,
+            `md:w-[calc(100vw-${isSidebarOpen ? "21.5" : "8"}rem)]`,
+          )}
```

Comment:
> The purpose of cn utility is more readability and avoiding the string interpolation

The following is more readable and should work
```suggestion
          className={cn(
            "transition-all duration-200",
            isSidebarOpen 
              ? "ease-out md:w-[calc(100vw-21.5rem)]" 
              : "ease-in md:w-[calc(100vw-8rem)]"
          )}
```

Reply:
https://tailwindcss.com/docs/detecting-classes-in-source-files#dynamic-class-names